### ISSUE TITLE
Discuss with original maintainer a plan to migrate existing data, issues or more

### ISSUE THREAD

**Issue by CloCkWeRX** [chef-rbenv/chef-rbenv#1]
The original repo is still going to attract a bunch of attention; so if we can get everything pointing here / issues turned off that might be handy.

**Comment by CloCkWeRX** [comment]
I spent a bit of time looking at this; and its annoying that the RiotGames rbenv cookbook is the only one on the supermarket.

I pushed an v0.7.3 release tag; but it'd be good to publish this somewhere people can get the fixes easily.

**Comment by josacar** [comment]
Maybe @fnichol can add a `DEPRECATED` section in the original cookbook's README.

**Comment by fnichol** [comment]
@CloCkWeRX If it makes sense, I could transfer fnichol/chef-rbenv to chef-rbenv/chef-rbenv which would maintain the git clone urls, issues, etc. The downside is that there would be some git work to take the work here back to the older repo and that this repo name would have to be renamed to something else (for the transfer to work anyway).

Would this straight move help things or muddy the waters further? @josacar's has a way to go as well with making it clear on the original repo

**Comment by josacar** [comment]
Agreed with @fnichol , we can replay all the work here later, but we will not lose track of pending issues or PRs
